test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -11,6 +11,23 @@
 public class UtilsTest {
 
 
+    @Test
+    public void elementsAreNull() throws Exception {
+
+        // empty list
+        assertNotNull();
+        
+        assertAreNull(null, null);
```

Comment:
Separate scenario should have their own comment to indicate what scenario it is.